media-libs/mesa/metadata.xml

```diff
@@ -23,6 +23,7 @@
 		<flag name="valgrind">Compile in valgrind memory hints</flag>
 		<flag name="vdpau">Enable the VDPAU acceleration interface for the Gallium3D Video Layer.</flag>
 		<flag name="vulkan">Enable Vulkan drivers</flag>
+		<flag name="vulkan-overlay">Enable vulkan-overlay-layer for vulkan stats"</flag>
```

Comment:
Stray `"` character at the end of description.
Also it would be nice if description briefly informed what vulkan-overlay-layer actually is, for whom it is useful etc.
(Based on other articles in internet, I conclude that it is useful only for developers of applications/drivers...)
Please use word `statistics`, not [informal](https://en.wiktionary.org/wiki/stats) `stats`.

Reply:
https://github.com/gentoo/gentoo/pull/11292